bolts/data/datasets/base.py

```diff
@@ -49,7 +49,8 @@ def logger(self) -> logging.Logger:
         return self._logger
 
     def log(self, msg: str) -> None:
-        self._logger.info(msg)
+        if self._logger is not None:
+            self._logger.info(msg)
```

Comment:
```suggestion
       self.logger.info(msg)
```

Reply:
can this be chained like this? `self.property.function_of_returned_value`?

Reply:
I don't see why not